CASMHMS-5522: NCN Remove changed stop REDS (#1530)
Cray-HPE/docs-csm: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_Data.md

    **Important**: Save the `ifnames` and `bmc_mac` information if you plan to add this NCN back at some time in the future.

> 1. Shutdown cray-reds:

Reviewer: ```suggestion
1. Shutdown `cray-reds`:
```

    Successfully removed x3000c0s26b0n0 - ncn-s004
    ```

    **NOTE:**

>     If workers have been removed and the worker count is currently at two, a time out restarting cray-bss can be ignored. For example, the following failure output from `remove_management_ncn.py` can be ignored.

Reviewer: ```suggestion
    If workers have been removed and the worker count is currently at two, a timeout restarting `cray-bss` can be ignored. For example, the following failure output from `remove_management_ncn.py` can be ignored.
```

>     ```screen

Reviewer: ```suggestion
    ```text
```

    Waiting for cray-bss to start.
    Do not kill this script. The wait will timeout in 10 minutes if bss does not fully start up.
    Ran:     kubectl -n services rollout status deployment cray-bss --timeout=600s
    Error:          Failed: 1
             stdout:
    Waiting for deployment "cray-bss" rollout to finish: 0 out of 3 new replicas have been updated...
    Waiting for deployment "cray-bss" rollout to finish: 1 out of 3 new replicas have been updated...
    Waiting for deployment "cray-bss" rollout to finish: 2 out of 3 new replicas have been updated...

             stderr:
    error: timed out waiting for the condition
    ```

> 1. Start cray-reds:

Reviewer: ```suggestion
1. Start `cray-reds`:
```
